packages/vue/src/composables/useAuth.ts

```diff
@@ -3,10 +3,13 @@ import { ref, reactive, Ref, watchEffect } from 'vue';
 import { getServiceFacade } from '@aws-amplify/ui';
 import { facade } from './useUtils';
 import { InterpretService } from '@/components';
-import { createSharedComposable } from '@vueuse/core';
 
 const service = ref() as Ref<InterpretService>;
-let useAuthenticatorValue = reactive({ ...facade, send: '', state: '' }) as any;
+const useAuthenticatorValue = reactive({
+  ...facade,
+  send: '' as unknown,
+  state: '' as unknown,
+}) as any;
```

Comment:
This needs types added in the future. If I remove the any it breaks a lot of the app, so this will need to be removed in a separate future PR, so `useAuthenticator` can have types.

Reply:
That'd be great!